sdk/core/azure-core/src/samples/java/com/azure/core/util/tracing/TracerJavaDocCodeSnippets.java

```diff
@@ -36,22 +38,27 @@ public void startTracingSpan() {
 
         // start a new tracing span with explicit parent, sets the request attributes on the span and sets the span
         // kind to client when process kind SEND
-        tracer.start("azure.eventhubs.send", sendContext, ProcessKind.SEND);
+        Context updatedSendContext = tracer.start("azure.eventhubs.send", sendContext, ProcessKind.SEND);
+        System.out.printf("Span returned in the context object: %s%n",
+            updatedSendContext.getData(OPENCENSUS_SPAN_KEY).get());
         // END: com.azure.core.util.tracing.start#string-context-processKind-SEND
 
         // BEGIN: com.azure.core.util.tracing.start#string-context-processKind-RECEIVE
         // start a new tracing span with explicit parent, sets the diagnostic Id (traceparent headers) on the current
         // context when process kind RECEIVE
-        Context updatedContext = tracer.start("azure.eventhubs.receive", traceContext, ProcessKind.RECEIVE);
-        System.out.printf("Diagnostic Id: {} %s%n", (String) updatedContext.getData(DIAGNOSTIC_ID_KEY).get());
+        Context updatedReceiveContext = tracer.start("azure.eventhubs.receive", traceContext,
+            ProcessKind.RECEIVE);
+        System.out.printf("Diagnostic Id: {} %s%n", (String) updatedReceiveContext.getData(DIAGNOSTIC_ID_KEY).get());
```

Comment:
Do you need to cast it to (String)? It'll do a .toString() on whatever object is returned from DIAGNOSTIC_ID_KEY.

Reply:
The {} placeholders are not required. This is only for slf4j